lib/licence_importer.rb

```diff
@@ -0,0 +1,81 @@
+require "common_licence_identifiers"
+
+class LicenceImporter
```

Comment:
> I think this class should be namespaced with "LicenceTransaction" to separate it from the other generic classes. ie. LicenceTransaction::LicenceImporter
I've also just seen the "importer" directory within "lib" so I think this class would fit there.
I think the LicenceTransaction::ImportIndustrySectors being created in PR https://github.com/alphagov/specialist-publisher/pull/2207 should/will also be moved there so that all of the import code is in one place.

Discussed async and have moved to namespaced directory.

> I think it would have helped with the review if the creation of the class had been broken into smaller commits so that some of the decisions could have been more easily explained and understood.

Noted, I've added more information to the commit message to explain the 'why'.